addons/website_sale/views/templates.xml

```diff
@@ -1162,6 +1162,18 @@
                                             <input type="text" name="vat" class="form-control" t-att-value="'vat' in checkout and checkout['vat']" />
                                         </div>
                                     </t>
+                                    <t t-else="1">
```

Comment:
no t-else, everything in t-if and t-att-readonly and t-att-title can_edit_vat and or